lib/content-services/src/lib/aspect-list/aspect-list.component.html

```diff
@@ -6,7 +6,8 @@
                     <mat-checkbox class="adf-aspect-list-check-button" [id]="'aspect-list-'+colIndex+'-check'"
                                   [checked]="nodeAspects?.includes(aspect?.entry?.id)"
                                   (click)="onCheckBoxClick($event)"
-                                  (change)="onChange($event, aspect?.entry?.id)">
+                                  (change)="onChange($event, aspect?.entry?.id)"
+                                  [aria-label]="'aspect-list-'+colIndex+'-check'">
```

Comment:
I don't think this label will make any sense for the screen reader

Reply:
I checked the 'title' works for assistive technology but title property doesn't  supporting to <mat-checkbox>. Please suggest if there is any other solution/approach.

Reply:
@rbahirsheth I think what Denys is trying to say is that label is correct solution but the label you proposed isn't much human-readable

Reply:
This issue is resolved. Now screen reader is reading checkbox label.